lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -15,14 +15,14 @@ defmodule Kernel.LexicalTracker do
   Returns all remotes linked to in this lexical scope.
   """
   def remotes(arg) do
-    :gen_server.call(to_pid(arg), :ets, @timeout)
-    |> :ets.match({{:mode, :'$1'}, :'$2'})
+    references = :gen_server.call(to_pid(arg), :references, @timeout)
+    for({{:mode, module}, mode} <- references, do: {module, mode})
```

Comment:
We don't need this for anymore! We can replace the ETS by two maps, one of them for the mode and another for the directives. :) This means we no longer need to use `{:mode, module}` as key!